system_status/requirements_for_system_status.txt

```diff
@@ -0,0 +1,4 @@
+requests==2.31.0
+SQLAlchemy==1.4.49
+psycopg2-binary==2.9.7
+git+https://github.com/cds-snc/notifier-utils.git@feat/add-system-status-utilities
```

Comment:
⚠️ This needs to be updated once [the utils PR ](https://github.com/cds-snc/notification-utils/pull/264) is merged.

Reply:
i approved!